zerver/tests/test_subs.py

```diff
@@ -1744,7 +1744,7 @@ def test_bulk_subscribe_MIT(self):
         # Make sure Zephyr mirroring realms such as MIT do not get
         # any tornado subscription events
         self.assert_length(events, 0)
-        self.assert_max_length(queries, 7)
+        self.assert_max_length(queries, 8)
```

Comment:
I thought we did something with the caching system so that tests should have the same database queries each time; why did that not work in this case?
(I'm just worried there will be like 10 more similar things).